tensorboard/uploader/proto/write_service.proto

```diff
@@ -332,6 +332,9 @@ message WriteBlobRequest {
   // CRC32C of the entire blob.  Required, to protect against data corruption.
   // This should be set only when finalize_object=True.
   fixed32 final_crc32c = 7;
+  // Final size in bytes of the entire blob.  Required in the first request
+  // to allow quota allocaion and management.
+  int64 final_size = 8;
```

Comment:
Old uploaders will appear to send `0` for this field. Are we prepared
for how the server will handle that (e.g., rejecting the requests
because the client tries to send a nonempty blob after “declaring” that
it would be of length 0)? As long as the only relevant uploaders are
nightly releases, breaking them is fine with me, just worth considering.

Reply:
renamed to blob_bytes.

No usages outside nightlies at present.  I'll discuss with David about enforcement issues.